tools/nl/embeddings/build_embeddings.py

```diff
@@ -101,23 +100,32 @@ def _write_intermediate_output(name2sv_dict: Dict[str, str],
   sv_list = sorted(list(sv2names.keys()))
   name_list = [';'.join(sorted(sv2names[v])) for v in sv_list]
 
+  # Write to preindex file
+  new_preindex_filepath = str(_get_input_folder() / '_preindex.csv')
+  new_preindex_df = pd.DataFrame(list(name2sv_dict.items()),
+                                 columns=['sentence',
+                                          'dcid']).sort_values(by='sentence')
+  new_preindex_df.to_csv(new_preindex_filepath, index=False)
+
   # Write to local_merged_filepath.
+  local_merged_filepath = str(_get_old_preindex_folder() /
+                              'sv_descriptions.csv')
   print(
       f"Writing the concatenated dataframe after merging alternates to local file: {local_merged_filepath}"
   )
   df_svs = pd.DataFrame({'dcid': sv_list, 'sentence': name_list})
   df_svs.to_csv(local_merged_filepath, index=False)
 
+  dup_names_filepath = str(_get_old_preindex_folder() / 'duplicate_names.csv')
```

Comment:
Just curious, do you plan to get rid of duplicate names (i.e., allow them and add multiple dcids)?

Reply:
Yes, right after this PR, since it will involve svdiff.